pkg/plugins/policies/core/matchers/testdata/matchedpolicies/dataplane-kind/simple-policy.policies.yaml

```diff
@@ -0,0 +1,24 @@
+# 01. policies using kind Dataplane selecting Dataplanes by labels
```

Comment:
test name can be more meaningful here, instead of "simple policy" we should probably name it "select all dpps" or something like that

Reply:
changed

Reply:
why did you remove the comments? :) I was expecting you're going to update them to be relevant

Reply:
added missing comments